lib/src/widgets/bottom_bar_with_sheet_item.dart

```diff
@@ -11,9 +11,10 @@ import 'package:provider/provider.dart';
 const defaultDuration = Duration(milliseconds: 500);
 
 // ignore: must_be_immutable
-class BottomBarWithSheetItem extends StatelessWidget {
+class BottomBarWithSheetItem<T> extends StatelessWidget {
   final String label;
-  final IconData icon;
+  final T icon;
```

Comment:
**T** is not the right type of parameter for this situation.

Anything can be passed for this type of parameter. But it should be an **iconData**.
It will be bad if someone confuses and puts a String in this field.